### PR TITLE
feat(feedback): adding frontend checkbox to include system info

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

This PR is the follow up of https://github.com/podman-desktop/podman-desktop/pull/10098 (which added the backend support) to collect the system-info of the host machine.

### Screenshot / video of UI

image

**demo**
https://github.com/user-attachments/assets/1de482ab-1b0b-4d68-8ac6-2ec57f48a526

> ℹ️ design not updated

### What issues does this PR fix or reference?

Part of https://github.com/podman-desktop/podman-desktop/issues/9585

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Comment by benoitf** [comment]
issue(blocking): checkbox is far away from the text

image

non blocking, follow-up: related to the other PR (follow-up), I guess we should use macOS rather than Darwin when doing the report. It was not obvious when reviewing the other PR but I think it'll be more meaningful to have macOS in the name
placeholder being:
image


image


issue:(follow-up or part of this PR)
Having the fields being completed by the link, I see one issue is that for example it doesn't set the version
image

I'm using .next version but then clicking on the link it selected 1.14.1 so I may open bugs for a wrong version

**Comment by axel7083** [comment]
> issue(blocking): checkbox is far away from the text

Fixed ✅ 

image

> non blocking, follow-up: related to the other PR (follow-up), I guess we should use macOS rather than Darwin when doing the report.

I created https://github.com/podman-desktop/podman-desktop/issues/10118 and assigned myself 

> issue:(follow-up or part of this PR)
> Having the fields being completed by the link, I see one issue is that for example it doesn't set the version

If it was working I would do as follow up, but this is not possible:

The following url:

https://github.com/podman-desktop/podman-desktop/issues/new?template=bug_report.yml&title=hello&version=1.13.2 

Do not allow us to set the version. I opened an issue on the GitHub report, the selected value is always the latest...
We could fix that by changing the dropdown to a text input..

**Review by benoitf — APPROVED** [review]
if you could update the screenshot before merging

**Comment by benoitf** [comment]
it seems the vertical align of the checkbox and the text is not the same as in other part of the UI, maybe needs to be checked before the merge

File: packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.spec.ts [hunk 1]
```diff
@@ -109,7 +115,6 @@ test('Expect Preview on GitHub button to be disabled if there is no title or des
     expect(preview).toBeEnabled();
   });
 });
-
```
Comment (gastoner) [inline packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.spec.ts]: Formated or missing new line?

**Comment by axel7083** [comment]
> it seems the vertical align of the checkbox and the text is not the same as in other part of the UI, maybe needs to be checked before the merge

This seems to originate inside the `Checkbox` component

https://github.com/podman-desktop/podman-desktop/blob/516922e2ef043c208d55e9921edda79f7798aa20/packages/ui/src/lib/checkbox/Checkbox.svelte#L29

image

Not something that can be addressed in this PR.

**Review by gastoner — APPROVED** [review]
LGTM, tested on Windows

File: packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.spec.ts (hunk 1 above)
Comment (axel7083) [reply packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.spec.ts]: yes! fixed that, nice catch 👍

**Pull request merged**